test/DynamoCoreTests/DSEvaluationViewModelTest.cs

```diff
@@ -115,8 +115,11 @@ public void AssertClassName(string guid, string className)
             string varname = GetVarName(guid);
             var mirror = GetRuntimeMirror(varname);
             Assert.IsNotNull(mirror);
```

Comment:
I'm not sure why we are checking for the runtime mirror obtained here from the `guid` in the first place. This check does not go with the purpose of this function, which is to verify the class name. Is it possible to refactor the `guid` check into another API?